### PR TITLE
What's new updates July 26 - Aug 2

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

This pull request updates the [What's New](https://devdocs.magento.com/whats-new.html) page with the documentation changes published since the last update on July 26.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,9 +4,55 @@ description: |
...
+updated: Fri Aug 2 12:05:50 2019
 entries:
-- description: Updated changes in the 2.3.2 schema of the [customer endpoint](https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.htm).
+- description: Added information about patch PRODSECBUG-2233 to the relevant release
+    notes for affected Magento versions.
+  versions: 2.2.8, 2.2.x, 2.3.0, 2.3.1, 2.3.x, 2.x
+  type: Major update
+  date: August 2, 2019
+  link: https://github.com/magento/devdocs/pull/5091
+- description: Added the `Backward Compatibility` section to the [Magento Definition
```
Comment (shrielenee) [inline _data/whats-new.yml]: DO sections go in back ticks for code formatting?

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,9 +4,55 @@ description: |
...
+  link: https://github.com/magento/devdocs/pull/5056
+- description: Added a new section about [changing the Blackfire account owner](https://devdocs.magento.com/guides/v2.3/cloud/project/project-integrate-blackfire.html#change-account-owner)
+    on a Magento Commerce (Cloud) project.
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: July 29, 2019
+  link: https://github.com/magento/devdocs/pull/5070
+- description: Updated the [Fastly Set up](https://devdocs.magento.com/guides/v2.3/cloud/cdn/configure-fastly.html)
+    instructions to remove the requirement to install the Fastly CDN module for Magento
+    2 in Magento Commerce Cloud environments. The latest version of the Fastly module
```
Comment (shrielenee) [inline _data/whats-new.yml]: Is the second sentence needed?

File: _data/whats-new.yml [hunk 3]
```diff
@@ -323,7 +369,7 @@ entries:
   type: New topic
   date: June 22, 2019
   link: https://github.com/magento/devdocs/pull/4428
-- description: Added the `isEmailAvailable` query to the GraphQL [customer endpoint](https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html).
+- description: Added the /`isEmailAvailable` query to the GraphQL [customer endpoint](https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html).
```
Comment (meker12) [inline _data/whats-new.yml]:
```suggestion
- description: Added the `isEmailAvailable` query to the GraphQL [customer endpoint](https://devdocs.magento.com/guides/v2.3/graphql/reference/customer.html).
```

Comment (shrielenee) [inline _data/whats-new.yml]: I don't know why the / is here? "/`isEmailAvailable`"

Comment (meker12) [reply _data/whats-new.yml]: fixed it. Thx

File: _data/whats-new.yml (hunk 2 above)
Comment (meker12) [reply _data/whats-new.yml]: Yes

File: _data/whats-new.yml (hunk 1 above)
Comment (meker12) [inline _data/whats-new.yml]:
```suggestion
- description: Added the `Backward Compatibility` section to the [Magento Definition
```

**Comment by meker12** [comment]
running tests

**Comment by dshevtsov** [comment]
@meker12 the pull request title says about updates since July 26, but in the description the date is July 15.

**Comment by meker12** [comment]
running tests

**Pull request merged**